WalletWasabi/Blockchain/Transactions/TransactionStore.cs

```diff
@@ -392,5 +395,38 @@ private async Task TryCommitToFileAsync(ITxStoreOperation operation)
 		}
 
 		#endregion Serialization
+
+		private void ThrowIfDisposed()
+		{
+			if (_disposed)
```

Comment:
If this class can be used by multiple threads, then even this variable should be guarded by a lock (e.g. `_disposeLock`).

Reply:
it is fine IMO.

Reply:
Are we talking about that _disposed might be outdated when read?

Reply:
Yes, you may read stale value (i.e. not correct value).

Reply:
added volatile.